tests/unit/models/test_model_interfaces.py

```diff
@@ -855,5 +834,9 @@ def test_squeeze_hyperparameters_when_param_at_edge_of_bounds() -> None:
     kernel.lengthscales = gpflow.Parameter(
         kernel.lengthscales, transform=tfp.bijectors.Sigmoid(low=lower, high=upper)
     )
-    squeeze_hyperparameters(kernel, 0.1)
+    squeeze_hyperparameters(kernel, alpha=0.1)
     npt.assert_array_almost_equal(kernel.lengthscales, [0.1 + 4e-2, 0.5 - 4e-2])
```

Comment:
Im not sure this test is doing anything. Do we not need the lengthscales to be closer to their bounds for the squeeze to do anything (as epsilon is 1e-7)?

Reply:
We apply the squeeze even if the parameters are within bounds.